src/index.html

```diff
@@ -5,8 +5,327 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <nav class="header__nav">
+        <a href="" class="header__logo">
+          <img src="images/logo.png" alt="MOYO">
+        </a>
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="" class="header__link header__link--is-active is-active">Apple</a>
+          </li>
+          <li>
+            <a href="" class="header__link">Samsung</a>
+          </li>
+          <li>
+            <a href="" class="header__link">смартфоны и телефоны</a>
+          </li>
+          <li>
+            <a
+              href=""
+              data-qa="hover"
+              class="header__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li>
+            <a href="" class="header__link">гаджеты</a>
+          </li>
+          <li>
+            <a href="" class="header__link">планшеты</a>
+          </li>
+          <li>
+            <a href="" class="header__link">фото</a>
+          </li>
+          <li>
+            <a href="" class="header__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-catalog">
+      <div class="card main-catalog__card" data-qa="card">
+        <div class="card__picture card__picture--margin"></div>
+        <div class="card__description card__description--margin">
+          <p class="card__description-text">
+            Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)
+          </p>
+        </div>
+        <div class="card__item-code card__item-code--margin">
+          Код товара: 195434
+        </div>
+
+        <div class="card__review card__review--margin">
+          <div class="stars card__stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__text-review">Отзывов: 5</div>
+
+          <div class="card__price card__price--margin">
+            <p class="card__price-text card__price-text--margin">Цена:</p>
+            <p class="card__price-sum card__price-sum--margin">69 999 грн</p>
+          </div>
+        </div>
+        <a
+          href="#"
+          class="card__button-buy card__button-buy--hover card__button-buy--margin"
+          data-qa="hover"
+        >
+          КУПИТЬ
+        </a>
+      </div>
+      <div class="card main-catalog__card" data-qa="card">
```

Comment:
Leave `data-qa="card"` attribute only for the first card. This will help with tests.